src/labels/CustomLabels.labels

```diff
@@ -1983,6 +1991,14 @@
         <shortDescription>Fixed Length Recurring Donation End Date must be valid</shortDescription>
         <value>End Date can&apos;t be in the future when Recurring Type is Fixed.</value>
     </labels>
+    <labels>
+        <fullName>RD2_EndDateMustBeValidForElevateRecurringDonation</fullName>
```

Comment:
Let's shorten the var name if possible, for example `RD2_EndDateElevateCannotBeInFuture`?